base-noprelude.cabal

```diff
@@ -35,10 +35,11 @@ source-repository head
     location: https://github.com/hvr/base-noprelude.git
 
 library
-    build-depends:       base ==4.14.3.0
+    build-depends:       base ==4.16.1.0
     default-language:    Haskell2010
-
-    -- re-exported modules copied from base-4.12.0.0's exposed-modules
+    -- re-exported modules copied from exposed-modules of `base.cabal`
+    -- as of https://gitlab.haskell.org/ghc/ghc @ eea96042f1e8682605ae68db10f2bcdd7dab923e 
+    -- (best match to base-4.16.0.0 docs)
```

Comment:
it seems that in some of these commits there are some `exposed-modules` from `base` that are missing here
e.g. in a1606be67e7ea64ccd41e978dfde54d9c5c7c8e6 I don't see:
- `GHC.Event.TimeOut`
- `GHC.IO.StdHandles`
- `GHC.IO.SubSystem`
- `GHC.Integer`
- `GHC.Integer.Logarithms`
- `GHC.IOPort`

nor some OS-specific ones (see at the bottom).

I'm mentioning this here because of this comment as well, because I think that it's easier to grab the list of modules from `hackage` itself, by visiting the:
> Package description (as included in the package)

link, aka:
```
https://hackage.haskell.org/package/base-<version>/base.cabal
```
for each `<version>` and looking for the `exposed-modules` there.

Reply:
Indeed! Fixed now, I think.

Reply:
just checked, AFAICT everything appears to be present.

I see you have also included the `reexported-modules` from `base`, good call :ok_hand: